Zend/zend_max_execution_timer.c

```diff
@@ -98,6 +109,16 @@ void zend_max_execution_timer_shutdown(void) /* {{{ */
 	if (err != 0) {
 		zend_strerror_noreturn(E_ERROR, errno, "Could not delete timer");
 	}
+
+	struct sigaction ign_sigrtmin, prev;
+	ign_sigrtmin.sa_handler = SIG_IGN;
+	ign_sigrtmin.sa_flags = 0;
+	sigemptyset(&ign_sigrtmin.sa_mask);
+
+	if(sigaction(SIGRTMIN, &ign_sigrtmin, &prev) < 0) {
```

Comment:
```suggestion
	if(sigaction(SIGRTMIN, &ign_sigrtmin, NULL) < 0) {
```

And the removal of the prev local variable.
The old action argument is nullable, which just means that the old action isn't stored.